README.md

```diff
@@ -10,6 +10,12 @@ Batarangle is a Google Chrome Dev Tools extension for debugging Angular 2 applic
 
 If you want to contribute or need help getting started, [join us on Slack](https://batarangle-slack.herokuapp.com).
 
+## Getting Extension
```

Comment:
@sumitarora can we mention that there are 2 ways of getting it?
Specifically, 1) If you just want to use it, download it from batarangle.io 
2) for more adventurous people there are instructions on how to build and pack it yourself. (we should mention that our CI process creates a package with the latests master on batarangle.io)